aiida/plugins/factories.py

```diff
@@ -55,14 +56,15 @@ def CalculationFactory(entry_point_name):
     :return: sub class of :py:class:`~aiida.engine.processes.calcjobs.calcjob.CalcJob`
     :raises aiida.common.InvalidEntryPointTypeError: if the type of the loaded entry point is invalid.
     """
+    from inspect import isclass
```

Comment:
The second import probably shouldn't be here. The rest looks good 👍

Reply:
sure, thanks :)